README.md

```diff
@@ -225,7 +224,9 @@ This is the backend API repository for TurLink. TurLink is a link shortener app
           "attributes": {
               "original": "testlink.com",
               "short": "tur.link/4a7c204baeacaf2c",
-              "user_id": 1
+              "user_id": 1,
+              "click_count": 1,
+              "last_click": "2024-08-28T12:34:56.789Z"
           }
```

Comment:
should we not change this datetime format to be slightly more readable? I guess if the frontend isn't displaying it anywhere we'll really only be using it to check for last_click for the inactive links feature

Reply:
also, we may not need to include this in the serializer at all if the frontend doesn't need this piece of information